Sources/Locker/Internal/DeviceList.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
+//  
+//
+//  Created by Zvonimir Medak on 08.10.2021..
+//
+
+import Foundation
+
+class DeviceList {
+    static let shared = DeviceList()
+
+    private(set) var api: URL = Constants.Apiary.deviceList
+
+    private init() {}
+}
```

Comment:
This class should be extended in a way to:
- read/write internal storage (we should store fetched devices so that we can avoid unneeded API calls)
- check if a device is already stored in the file/list
- internally fetch/update file with new devices